integration/spark/src/main/scala/org/apache/spark/sql/execution/command/schema/CarbonAlterTableColRenameDataTypeChangeCommand.scala

```diff
@@ -258,7 +262,7 @@ private[sql] case class CarbonAlterTableColRenameDataTypeChangeCommand(
       }
     } catch {
       case e: Exception =>
-        if (carbonTable != null) {
+       if ( carbonTable != null) {
```

Comment:
there has a blank between the '(' and carbonTable

Reply:
I have modified code according to you suggestion.